articles/introduction-to-lxd-container-manager/index.md

```diff
@@ -0,0 +1,38 @@
+### Introduction
+LXD is a highly effective container manager. It provides an individual experience same as that offered by virtual machines. LXD systems are unique because they operate with Linux containers. This technology was introduced by Canonical Ltd alongside other contributors. LXD is supported by numerous Linux distributions. One of the significant pillars of LXD is a powerful REST API. 
```

Comment:
You could have this paragraph as your second paragraph.

In your first paragraph, you must talk about:
- What a container is
- What a container manager is
- What virtual machines are
- Give a little abstract intro about LXD (if possible add link)

Reply:
Does `LXD` have an abbreviation? If so, please abbreviate it once, at the first usage. Specify the acronym along with it.

Reply:
LXD does not have an abbreviation